stack-lts-2.yaml

```diff
@@ -0,0 +1,12 @@
+resolver: lts-2.22
+packages:
+- '.'
+- location: example/
+  extra-dep: true
```

Comment:
it needs aeson >=0.9 because the serialisation of `UTCTime` is changed, but having `aeson-0.9` with lts-2.22 is tricky.

I could add a `resolver: ghc-7.8.4` based stack.yaml with newest deps - then the tests could be enabled.